spec/setup_acceptance_node.pp

```diff
@@ -11,3 +11,15 @@
 package { $dig_package:
   ensure => installed,
 }
+
+if $facts['os']['name'] == 'CentOS' {
+  if $facts['os']['release']['major'] == "7" {
+    package { 'epel-release':
+      ensure => present,
+    }
+  } elsif $facts['os']['release']['major'] == "8" {
+    package { 'centos-release-ansible-29':
+      ensure => present
+    }
```

Comment:
Should we also use this on EL7? I think it works in the same way, right?

Reply:
Oh, I totally forgot this also exists on EL7, yeah, better.